src/wallet/rpcwallet.cpp

```diff
@@ -1359,6 +1359,23 @@ static void ListTransactions(interfaces::Chain::Lock& locked_chain, CWallet* con
     }
 }
 
+static const std::string TransactionDescriptionString()
+{
+    return "    \"confirmations\": n,                        (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
+           "                                                       transaction conflicts with the block chain.\n"
+           "    \"generated\": xxx,                          (bool) Only returns true if transaction only input is a coinbase one, otherwise false.\n"
+           "    \"trusted\": xxx,                            (bool) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
+           "    \"blockhash\": \"hashvalue\",                (string) The block hash containing the transaction.\n"
```

Comment:
nit, try to have it aligned with surrounding lines.

Reply:
You mean `blockhash` compare to `trusted` ?

Reply:
I think the first commit (moveonly) should actually be moveonly (via `git diff --color-moved=dimmed-zebra`).

Otherwise the rendered whitespace will be messed up. (You can get the help text with the `help listtransactions` rpc)